### ISSUE TITLE
Fix generating empty spectrums

### ISSUE THREAD

**Issue by lionel42** [matchms/matchms#588]
**Describe the bug**

The following code generate an empty spectrum

```
s = Spectrum(np.array([1.]), np.array([1.]))
processing = SpectrumProcessor(
    [
        lambda spec: select_by_mz(spec, 20, 30)
    ]
)
processing.process_spectrums([s])

```

 It raises an error when trying to print the spectra, but I assume this can casue other problems. (I found one while trying to compute scores using this spectra.

```
ValueError: min() arg is an empty sequence
```

**To Reproduce**
Code above.

**Expected behavior**
It should not break, I guess empty spectrum is find in some cases, when processing data.

**Desktop (please complete the following information):**
 - OS: Windows
 - Version python3.11.2 ,  matchms:0.24.0

**Comment by florian-huber** [comment]
Thanks for the issue @lionel42 .
This happens because `Spectrum` cannot yet handle empty peak arrays. I will look into this.

**Comment by florian-huber** [comment]
Should be fixed with #598.
